File: SMM.IServices/Models/Project/HashtagModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SMM.IServices.Models.Project
{
    public class HashtagModel
    {
        /// <summary>
        /// Ид
        /// </summary>
        public int Id { get; set; }
        /// <summary>
        /// Название хэштэга
        /// </summary>
        public string Title { get; set; }
        /// <summary>
        /// Ид проекта
        /// </summary>
        public int ProjectId { get; set; }
    }
}

File: SMMService/asserts/js/Forms/Project/HashTagModel.js
var Project = Project || {};

(function () {

    Project.HashTagModel = function (theParams) {
        theParams = theParams || {};
        this.Id = ko.observable(theParams.Id || 0);
        this.Title = ko.observable(theParams.Title || "");
        this.ProjectId = ko.observable(theParams.ProjectId || "");
        return this;
    };

    /**
    * Определяем конструктор
    */
    Project.HashTagModel.prototype.constructor = Project.HashTagModel;

    Project.HashTagModel.prototype.GetData = function () {
        return {
            Id: this.Id(),
            Title: this.Title(),
            ProjectId: this.ProjectId()

        }
    }

})();